.travis.yml

```diff
@@ -1,35 +1,37 @@
 language: go
 
 go:
-  - 1.4
   - 1.5
   - tip
 
+env:
+  - GO15VENDOREXPERIMENT=1
+
 before_install:
   - go get github.com/golang/lint/golint
   - go get golang.org/x/tools/cmd/vet
   - go get github.com/fzipp/gocyclo
   - go get github.com/mattn/goveralls
 
-  # get our dependencies
-  - go get -t ./...
-
 install:
   # make sure stuff actually builds
   - go build
 
 script:
+  # define our own packages so that go does not
+  # try and build our vendored deps
+  - pkgs=(. ./vow)
```

Comment:
We can actually use `go list ./... | grep -v 'vendor'` in case we add more packages.